### PR TITLE
Standardize on Node 14

### PR DESCRIPTION
**Pull request by mchelen-gov**
Node 14 is the most recent LTS release https://nodejs.org/en/about/releases/

- [x] update docker config
- [x] update gh actions
- [x] add nvm config
- [x] document nvm usage

### CODE REVIEW

**Comment by benwarfield-usds** [comment]
I have no objections to this but if anybody tries to merge it before next week I might lose it just a little. 😝

File: frontend/Dockerfile.dev [hunk 1]
```diff
@@ -13,4 +13,4 @@ RUN npm install
 
 COPY . /frontend/app
 
-CMD ["npm", "start"]
\ No newline at end of file
+CMD ["npm", "start"]
```
Comment (benwarfield-usds) [inline frontend/Dockerfile.dev:16]: 😆

**Comment by benwarfield-usds** [comment]
@mchelen-gov if you don't mind rebasing this to take care of the README conflicts, that would be awesome—otherwise, one of us will doubtless make appropriate arrangements sometime this week.

Thanks for doing this!

File: frontend/Dockerfile.dev (hunk 1 above)
Comment (mchelen-gov) [reply frontend/Dockerfile.dev:16]: vscode format on save ftw! :grin:

**Comment by mchelen-gov** [comment]
@benwarfield-usds Ok I took a crack at resolving merge conflict, lmk if it looks good! I think GH can squash commits on PR merge, if not I'm happy to do that.

File: frontend/Dockerfile.dev (hunk 1 above)
Comment (benwarfield-usds) [reply frontend/Dockerfile.dev:16]: Oh, I assumed vi, because I'm old. 👴

**Review by dmethvin-gov — APPROVED** [review]

File: frontend/Dockerfile.dev [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Dockerfile for local development
+<<<<<<< HEAD
```
Comment (TomNUSDS) [inline frontend/Dockerfile.dev:2]: Merge conflict resolve needed

Comment (benwarfield-usds) [reply frontend/Dockerfile.dev:2]: Oh crap, didn't see that comment. We can fix it in post. 😁

**Pull request merged**